yt_dlp/extractor/weyyak.py

```diff
@@ -0,0 +1,50 @@
+import re
+
+from .common import InfoExtractor
+from ..utils import traverse_obj
+
+
+class WeyyakIE(InfoExtractor):
+    _VALID_URL = r'https?://weyyak\.com/(?P<lang>\w+)/player/(?P<type>episode|movie)/(?P<id>\d+)(?:/.+)?'
+    _TESTS = [
+        {
+            'url': 'https://weyyak.com/en/player/episode/1341952/Ribat-Al-Hob-Episode49',
+            'md5': '0caf55c1a615531c8fe60f146ae46849',
+            'info_dict': {
+                'id': '1341952',
+                'ext': 'm3u8',
+                'title': 'Ribat Al Hob',
+                'duration': 2771,
+            },
+        }
+    ]
+
+    def _real_extract(self, url):
+        match = self._match_valid_url(url)
+
+        # Extract the video id, language and type from the url
+        _id = match.group('id')
+        _lang = match.group('lang')
+        _type = match.group('type')
```

Comment:
Don't use single use variable
```suggestion
        _id, _lang, _type = self._match_valid_url(url).group('id', 'lang', 'type')
```

Reply:
```suggestion
        display_id, lang, type_ = self._match_valid_url(url).group('id', 'lang', 'type')
```